run_action.py

```diff
@@ -181,6 +181,24 @@ def unescape_chars(s):
 
         return s
 
+    def markdown_url(label, url):
+        return f"[{label}]({url})"
+
+    def diagnostic_name_visual(diagnostic_name):
+        visual = f"**{markdown(diagnostic_name)}**"
+
+        try:
+            first_dash_idx = diagnostic_name.index("-")
+        except ValueError:
+            return visual
+
+        namespace = diagnostic_name[:first_dash_idx]
+        check_name = diagnostic_name[first_dash_idx + 1 :]
+        return markdown_url(
+            visual,
+            f"https://clang.llvm.org/extra/clang-tidy/checks/{namespace}/{check_name}.html",
```

Comment:
Hi @fandreuz what do you think about URL encoding the `namespace` and `check_name` URL parts here? Yes, based on the current rule name syntax, this is not required _for now_, but who knows what will happen in the future.

Reply:
Sure thing: eddbb392320db6c1024614e8a1f381c00946071d